ecommerce/src/api/FetchCart.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import { useApp } from "../context/App-context";
+import toast from "react-hot-toast";
+import axios from "axios";
+
+export default async function CartLoadHandler() {
+  const { state, dispatch } = useApp();
+  const userName = state.userName;
+  useEffect(async()=>{
+    await axios({
+        method: "get",
+        url: `https://jainwin.herokuapp.com/cart/${userName}`
+      })
+        .then(function (response) {
+          toast.success("cart loaded succesfully");
+          const { cart } = response.data;
+          dispatch({ type: "CARTUPDATE", payload: cart.itemsInCart });
+        })
+        .catch(function (error) {
+          console.log(error.response);
+        //   toast.error("");
+        });
+  },[])
```

Comment:
This does not seem right to create a component without a return statement, we can have discussion over it why you used it